templates/zerver/integrations/include/generate-integration-url.md

```diff
@@ -1,9 +1,19 @@
-Generate the URL for your integration by clicking the **link** (<i
-class="fa fa-link"></i>) icon on the profile card for the bot you
-created.
+Generate the URL for your {{ integration_display_name }} integration,
+with the stream (and topic) you want notifications sent to, by [viewing
+the bot you created](/help/view-your-bots) and clicking the **link**
+(<i class="fa fa-link"></i>) icon on the bot's profile card.
+
+The generated URL will be something like:
+
+{!webhook-url.md!}
+
+*To manually create the URL for an incoming webhook integration,
+see [specification details][incoming-webhook-urls].*
```

Comment:
"see the specification" feels more natural to me.

Reply:
Updated version: ![image](https://github.com/zulip/zulip/assets/2746074/b455455e-d758-4f54-8d87-891f5fdf021c)